sonic_platform_base/sfp_base.py

```diff
@@ -83,45 +83,45 @@ def get_transceiver_threshold_info(self):
         ========================================================================
         keys                       |Value Format   |Information
         ---------------------------|---------------|----------------------------
-        Temperature High Alarm     |FLOAT          |High Alarm Threshold value,
+        temphighalarm              |FLOAT          |High Alarm Threshold value,
                                    |               |of temperature in Celsius.
```

Comment:
I think it would be fine to keep the Information field on one line instead of splitting into two. The lines might be a bit long, but it should be around 100 chars, so I think it would be fine, and it would make the list shorter.

Reply:
fixed.